docker/README.md

```diff
@@ -0,0 +1,81 @@
+# Garage Docker
```

Comment:
I think this is more appropriately called "Using Docker to run garage"

Reply:
Done